src/set_theory/zfc/basic.lean

```diff
@@ -219,6 +221,8 @@ instance : is_empty (type (∅)) := pempty.is_empty
 
 @[simp] theorem mem_empty (x : pSet.{u}) : x ∉ (∅ : pSet.{u}) := is_empty.exists_iff.1
 
+@[simp] theorem empty_to_set : to_set ∅ = ∅ := by simp [to_set]
```

Comment:
```suggestion
@[simp] theorem to_set_empty : to_set ∅ = ∅ := by simp [to_set]
```

Reply:
(and throughout)

Reply:
I don't think there's an actual rule here, is there? It's a special case of the "does dot notation go before or after" conundrum. I've mostly used `X_to_Y` in the past...